<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEvent</class>
 <widget class="QDialog" name="EditEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nouvel évènement</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="HLayoutEvent">
     <item>
      <widget class="QLabel" name="LabelEvent">
       <property name="text">
        <string>Annonce :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineeditEvent">
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutTimecode">
     <item>
      <widget class="QLabel" name="LabelTimeCode">
       <property name="text">
        <string>Time code :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="EditTimeCode">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>HH:mm:ss</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="HSpacerTimecode">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="BoxAnnouncement">
     <property name="title">
      <string>Annonce vocale</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="HLayoutSound">
        <item>
         <widget class="QLabel" name="LabelSound">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Son :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComboSound"/>
        </item>
        <item>
         <widget class="QLabel" name="LabelSoundNotExisting">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Le son n'existe pas !</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="HSpacerSound">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLayoutFinalCountdown">
        <item>
         <widget class="QLabel" name="LabelFinalCountdown">
          <property name="text">
           <string>Décompte final :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="SpinboxFinalCountdown">
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="specialValueText">
           <string>Aucun</string>
          </property>
          <property name="suffix">
           <string> secondes</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="HSpacerFinalCountdown">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="BoxAnnouncement_2">
        <property name="title">
         <string>Rappels de l'annonce</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="HLAyoutNewAnnouncement">
           <item>
            <widget class="QLabel" name="LabelNewAnnouncement">
             <property name="text">
              <string>Nouveau rappel :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="EditAnnouncementTime"/>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonAddAnnouncement">
             <property name="text">
              <string>Ajouter</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="HSpacerAnnouncement">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonRemoveAnnouncement">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="ListAnnouncement"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <spacer name="HSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
